src/Components/CreateImageWizardV2/steps/Review/ReviewStepTextLists.tsx

```diff
@@ -50,9 +52,11 @@ import {
   selectGcpShareMethod,
   selectPackages,
   selectRegistrationType,
+  selectFileSystemConfiguration,
+  selectFileSystemPartitionMode,
 } from '../../../../store/wizardSlice';
 import { toMonthAndYear } from '../../../../Utilities/time';
-import { MajorReleasesLifecyclesChart } from '../../../CreateImageWizard/formComponents/ReleaseLifecycle';
+import { MajorReleasesLifecyclesChart } from '../ImageOutput/ReleaseLifecycle';
```

Comment:
Now I see why there is the change to the ReleaseLifecycle. This is great catch! Could you please open a separate PR for this one so the changes are not hidden in the FSC one?

Reply:
sure